src/Rules/PropertyConditions/BoundingRectangle.cs

```diff
@@ -52,7 +52,6 @@ private static bool HasCorrectDataFormat(IA11yElement e)
         private static bool IsBoundingRectangleEmpty(IA11yElement e)
         {
             if (e == null) throw new ArgumentNullException(nameof(e));
-            if (e.BoundingRectangle == null) throw new ArgumentException(ErrorMessages.ElementBoundingRectangleNull, nameof(e));
 
             // Because IA11yElement.BoundingRectangle will return an empty rectangle when the property value is null,
```

Comment:
Since BoundingRectangle can never be null, should this comment also be removed?

Reply:
The way I read this comment, it seems to be saying we can't null check `IA11yElement.BoundingRectangle` here, which is correct. With that interpretation, the comment didn't make sense before this PR because we were attempting to null check, and with this PR now does make sense. Happy to remove it if you read the comment differently though!

Reply:
Might be best to run this by @RobGallo to see if he remembers the intent of the comment, then either reword or remove, as appropriate.

Reply:
Removed the new line but left code comment in based on Rob's PR comment.